README.md

```diff
@@ -36,6 +36,7 @@ $ docker run -it -p 8080:8080 -e DEBUG="helia-http-gateway" helia
 | Variable | Description | Default |
 | --- | --- | --- |
 | `DEBUG` | Debug level | `''`|
+| `FASTIFY_DEBUG` | Debug level for fastify's logger | `''`|
```

Comment:
separate env var for enabling fastify logging, since we have our own logging using `debug` with `DEBUG` already.